integration/emr/alluxio-emr.sh

```diff
@@ -685,6 +685,10 @@ IN
       fi
 
       if [[ "${sync_list}" ]]; then
+        until ${ALLUXIO_HOME}/bin/alluxio fsadmin report
```

Comment:
what do you think about always waiting after starting so any future additional alluxio commands will be safe to run? so i propose to move this immediately after register_backup_on_shutdown block. the bootstrapping shouldn't take any longer since workers still have to start and this is on master

Reply:
Sounds good, it may be clearer to do that